pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/load_redis.py

```diff
@@ -0,0 +1,46 @@
+"""Load data into Redis so we can use it as a feature store."""
+import redis
+import pandas as pd
+import json
+
+def upload():
+	r = redis.Redis(host="localhost", port=6379, db=0)
+
+	df = pd.read_csv("~/Desktop/flights_sub.csv")
+
+	print("Uploading data to feature store")
+
+	pipeline = r.pipeline()
+	i = 0
+
+	# Upload data to feature store
+	for row in df.iterrows():
```

Comment:
It'll be fun to have a "data loading" benchmark as well.

Reply:
For sure, it's super slow here.